galasa-parent/dev.galasa.framework.api.openapi.servlet/src/main/java/dev/galasa/framework/api/openapi/servlet/OpenApiServlet.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.openapi.servlet;
+
+import java.io.IOException;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ServiceScope;
+
+import dev.galasa.framework.api.common.BaseServlet;
+import dev.galasa.framework.api.common.Environment;
+import dev.galasa.framework.api.common.EnvironmentVariables;
+import dev.galasa.framework.api.common.SystemEnvironment;
+import dev.galasa.framework.api.openapi.servlet.routes.OpenApiRoute;
+
+@Component(service = Servlet.class, scope = ServiceScope.PROTOTYPE, property = {
+    "osgi.http.whiteboard.servlet.pattern=/openapi/*" }, name = "Galasa OpenAPI servlet")
```

Comment:
are we insisting on the slash from the user in the URL ? That would be bad...

Reply:
No, the trailing `/` isn't required, an auth filter unit test was already added to exercise this